octodns/provider/route53.py

```diff
@@ -550,14 +550,25 @@ def _healthcheck_measure_latency(self, record):
             .get('healthcheck', {}) \
             .get('measure_latency', True)
 
+    def _healthcheck_request_interval(self, record):
+        interval = record._octodns.get('route53', {}) \
+            .get('healthcheck', {}) \
+            .get('request_interval', 10)
+        if (interval in [10, 30]):
+            return interval
+        else:
+            raise Exception('route53.healthcheck.request_interval '
+                            'parameter must be either 10 or 30.')
```

Comment:
Going the `raise` route it'd be preferable to create a specific exception to throw rather than the generic `Exception`. For example down below in the test it's catching `Exception` which would handle any problem, not just this one raising. It can be as simple as:

```python
class Route53ProviderException(Exception):
    pass
```

and then `s/Exception/Route53ProviderException/`.